polaris-react/src/components/Modal/components/CloseButton/CloseButton.tsx

```diff
@@ -1,29 +1,18 @@
 import React from 'react';
-import {MobileCancelMajor, CancelMajor} from '@shopify/polaris-icons';
+import {CancelMajor} from '@shopify/polaris-icons';
 
-import {classNames} from '../../../../utilities/css';
 import {useI18n} from '../../../../utilities/i18n';
-import {useFeatures} from '../../../../utilities/features';
-import {Icon} from '../../../Icon';
 import {Button} from '../../../Button';
 
-import styles from './CloseButton.scss';
-
 export interface CloseButtonProps {
   pressed?: boolean;
-  titleHidden?: boolean;
```

Comment:
Removed `titleHidden` prop because it was being used in the custom close button rendered without the flag.
Also removed the `CloseButton.scss` file because custom close button no longer rendered.